llms/anthropic/anthropicllm.go

```diff
@@ -262,36 +262,34 @@ func handleHumanMessage(msg llms.MessageContent) (anthropicclient.ChatMessage, e
 	return anthropicclient.ChatMessage{}, errors.New("invalid content type for human message")
 }
 
-type ToolUse struct {
-	Type  string `json:"type"`
-	ID    string `json:"id"`
-	Name  string `json:"name"`
-	Input string `json:"input"`
-}
-
 func handleAIMessage(msg llms.MessageContent) (anthropicclient.ChatMessage, error) {
```

Comment:
Can you follow on with a test covering this, there's enough going on here.

Reply:
Sure no problem, always happy to have more tests